deployer.rb

```diff
@@ -108,4 +119,16 @@ def slack_client
     @slack_client = Slack::Client.new
   end
 
+  def commit_messages(pull_request)
+    name = repo_name(pull_request)
+    number = pull_request_number(pull_request)
+    commits = client.pull_request_commits(name, number)
+    commits.collect do |c|
```

Comment:
I had to lookup the difference between `collect` and `map`... turns out they're the same, haha